editor/editor_help.cpp

```diff
@@ -2061,6 +2062,24 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 
 		if (brk_pos > pos) {
 			String text = bbcode.substr(pos, brk_pos - pos);
+			if (codeblock_tag && code_indent_type == 0) {
+				// Replace the code block's space indentation with tabulation.
```

Comment:
```suggestion
				// Replace the code block's space indentation with tabs.
```
"tabulation" means creating a list